game/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cubby_server_game"
+version = "0.1.0"
+authors = ["Chris Ohk <utilforever@gmail.com>", "Seungoh Han <so.h4ns@gmail.com>", "Jaeyong Sung <jaeyong0201@gmail.com>", "Seungbin Oh <sboh1214@gmail.com>"]
```

Comment:
We should add new members next time. 📝

Reply:
Added Minchurl Shin <minchurl001@gmail.com>